src/main/java/uk/gov/hmcts/reform/em/stitching/domain/Bundle.java

```diff
@@ -225,4 +230,37 @@ public String toString() {
                 + this.hasTableOfContents + ", hasCoversheets=" + this.hasCoversheets + ", hasFolderCoversheets="
                 + this.hasFolderCoversheets + ")";
     }
+
+    @Transient
+    public Integer getSubtiles(SortableBundleItem container, Map<BundleDocument, File> documents) {
```

Comment:
I think this method should be getSubtitles and also it should probably have a unit test